build.gradle

```diff
@@ -15,7 +15,7 @@ buildscript {
             'guava': '27.1',
             'findbugs': '3.0.2',
             'joda': '2.10.1',
-            'jackson': '2.14.1',
+            'jackson': '2.14.2',
```

Comment:
If you're going to start bumping dependencies, you should probably do them all (in a separate PR), but if you're going to bump just this one you should probably bump to `2.14.3`

Reply:
Done. The only ones changed relate to migrating to Gradle over 8.x